src/pages/homePage/component_homePage/InjectionSite.tsx

```diff
@@ -0,0 +1,591 @@
+
+  const handleFirstPageButtonClick = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    onPageChange(event, 0);
+  };
+
+  const handleBackButtonClick = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    onPageChange(event, page - 1);
+  };
+
+  const handleNextButtonClick = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    onPageChange(event, page + 1);
+  };
+
+  const handleLastPageButtonClick = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    onPageChange(event, Math.max(0, Math.ceil(count / rowsPerPage) - 1));
+  };
+
+  return (
+    <Box sx={{ flexShrink: 0, ml: 2.5 }}>
+      <IconButton
+        onClick={handleFirstPageButtonClick}
+        disabled={page === 0}
+        aria-label="first page">
+        {theme.direction === 'rtl' ? <LastPageIcon /> : <FirstPageIcon />}
+      </IconButton>
+      <IconButton
+        onClick={handleBackButtonClick}
+        disabled={page === 0}
+        aria-label="previous page">
+        {theme.direction === 'rtl' ? (
+          <KeyboardArrowRight />
+        ) : (
+          <KeyboardArrowLeft />
+        )}
+      </IconButton>
+      <IconButton
+        onClick={handleNextButtonClick}
+        disabled={page >= Math.ceil(count / rowsPerPage) - 1}
+        aria-label="next page">
+        {theme.direction === 'rtl' ? (
+          <KeyboardArrowLeft />
+        ) : (
+          <KeyboardArrowRight />
+        )}
+      </IconButton>
+      <IconButton
+        onClick={handleLastPageButtonClick}
+        disabled={page >= Math.ceil(count / rowsPerPage) - 1}
+        aria-label="last page">
+        {theme.direction === 'rtl' ? <FirstPageIcon /> : <LastPageIcon />}
+      </IconButton>
+    </Box>
+  );
+}
+
+const InjectionSites = () => {
+  const formSchema = Yup.object().shape({
+    provinceId: Yup.number(),
+    districtId: Yup.number(),
+    wardId: Yup.number()
+  });
+
+  const validationOpt = {
+    resolver: yupResolver(formSchema)
+  };
+
+  const { register, handleSubmit, watch, control } =
+    useForm<Inputs>(validationOpt);
+
+  const onSubmit = () => {};
+  const provinceId = watch('provinceId');
+  const districtId = watch('districtId');
+
+  const filterDistricts: District[] = React.useMemo(() => {
+    return districts.filter((district) => district.provinceId === provinceId);
+  }, [provinceId]);
+
+  const filterWards: Ward[] = React.useMemo(() => {
+    return wards.filter((ward) => ward.districtId === districtId);
+  }, [districtId]);
+
+  const findProvince = (IDsearch: Number) => {
+    let name;
+    provinces.find((element: Province) => {
+      if (element.id === IDsearch) {
+        name = element.name;
+      }
+    });
+    return name;
+  };
+
+  const findDistrict = (IDsearch: Number) => {
+    let name;
+    districts.find((element: District) => {
+      if (element.id === IDsearch) {
+        name = element.name;
+      }
+    });
+    return name;
+  };
+
+  const findWard = (IDsearch: Number) => {
```

Comment:
```suggestion
  const findWard = (wardId: Number) => {
```

Viết theo đúng convention camelCase
Chú ý cách đặt tên có ý nghĩa